src/pybind/mgr/cephadm/module.py

```diff
@@ -1617,7 +1635,12 @@ def _run_cephadm(self, host, entity, command, args,
             # this is a misleading exception as it seems to be thrown for
             # any sort of connection failure, even those having nothing to
             # do with "host not found" (e.g., ssh key permission denied).
-            raise OrchestratorError('Failed to connect to %s (%s).  Check that the host is reachable and accepts connections using the cephadm SSH key' % (host, addr)) from e
+            user = 'root' if self.mode == 'root' else 'cephadm'
+            msg = f'Failed to connect to {host} ({addr}).  ' \
+                  f'Check that the host is reachable and accepts connections using the cephadm SSH key\n' \
+                  f'you may want to run: \n' \
+                  f'> ssh -F =(ceph cephadm get-ssh-config) -i =(ceph config-key get mgr/cephadm/ssh_identity_key) {user}@{host}'
```

Comment:
@sebastian-philipp 
```
node1:~ # ssh -F =(ceph cephadm get-ssh-config) -i =(ceph config-key get mgr/cephadm/ssh_identity_key) root@master
-bash: syntax error near unexpected token `('
```

Reply:
Nevermind, I was using `cephadm` from master

Reply:
right. `get-ssh-config` was the missing piece in order to make this work.